src/components/widgets/ContentWidget.tsx

```diff
@@ -0,0 +1,20 @@
+import { useNavigation } from '@react-navigation/core';
+import React from 'react';
+
+import { Icon, texts } from '../../config';
+import { WidgetProps } from '../../types';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const ContentWidget = ({ additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+
+  return additionalProps?.content?.map((content) => (
+    <DefaultWidget
```

Comment:
as a default widget is rendered, i think "DefaultWidget" make more sense than "ContentWidget"? what should content stand for?

Reply:
I updated the name of the widget to `ServiceWidget`, I think it is more understandable this way.

Reply:
Isn't it possible to use the default widget directly? All other widgets like weather are special widgets using the default widget as a base, right? But if we have default functionality, we could use default widget directly? I hope that we do not need a wrapper widget to return default widget for using it?

Reply:
We talked over Slack and decided to keep it as a wrapper-widget. The name of the widget has been updated to `CustomWidget`.